proposals/0110-distingish-single-tuple-arg.md

```diff
@@ -1,10 +1,10 @@
 # Distinguish between single-tuple and multiple-argument function types
 
 * Proposal: [SE-0110](0110-distingish-single-tuple-arg.md)
-* Authors: Vladimir S., Austin Zheng
-* Review Manager: [Chris Lattner](http://github.com/lattner)
-* Status: **Implemented (Swift 4)**
-* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2016-July/000215.html)
+* Authors: Vladimir S., [Austin Zheng](https://github.com/austinzheng)
+* Review Manager: [Chris Lattner](https://github.com/lattner), [Doug Gregor](https://github.com/DougGregor)
+* Status: **Implemented (Swift 4 beta)**, **Reverted (Swift 4)**
+* Decision Notes: [Implementation Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2016-July/000215.html), [Reversion Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2017-June/000386.html)
```

Comment:
Only a *portion* of SE-0110 was rolled back in apple/swift#10027 and apple/swift#10414, so you might want to change to:

```diff
-* Status: **Implemented (Swift 4 beta)**, **Reverted (Swift 4)**
-* Decision Notes: [Implementation Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2016-July/000215.html), [Reversion Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2017-June/000386.html)
+* Status: **Deferred**
+* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2016-July/000215.html), [Additional Commentary](https://lists.swift.org/pipermail/swift-evolution-announce/2017-June/000386.html)
```

A status from [process.md](https://github.com/apple/swift-evolution/blob/master/process.md#proposal-states) is required, so that the proposal appears on the [status page](https://apple.github.io/swift-evolution/). I don't know if the status page can handle more than one review manager.